include/evmc/evmc.h

```diff
@@ -813,19 +813,26 @@ enum evmc_revision
     /**
      * The Istanbul revision.
      *
-     * The spec draft: https://eips.ethereum.org/EIPS/eip-1679.
+     * https://eips.ethereum.org/EIPS/eip-1679
      */
     EVMC_ISTANBUL = 7,
 
     /**
      * The Berlin revision.
      *
-     * The spec draft: https://eips.ethereum.org/EIPS/eip-2070.
+     * https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/berlin.md
```

Comment:
Still don't get why the EIPs are deprecated and now there's a volatile place for the spec, i.e. this link has changed twice already since it has been added to the EIP 😬

Reply:
This looks bad to me too. I think we should ask for shorter permanent links.